docs/transparent-validation/twonkie/introduction.md

```diff
@@ -20,56 +20,70 @@ Links:
 
 ### Preparation
 
+- USB Micro-B cable
+- Twonkie v2.0
+- USB-PD power meter
+
 For flashing and using the Twonkie, you will need to install `dfu-util` and
 `minicom`. You can find them in your Linux distro's repositories:
 
 === "Ubuntu / Debian"
 
     ```bash
-    $ sudo apt install dfu-util minicom
+    sudo apt install dfu-util minicom
     ```
 
 === "Fedora"
 
     ```bash
-    $ sudo dnf install dfu-util minicom
+    sudo dnf install dfu-util minicom
     ```
 
 === "Arch"
 
     ```bash
-    $ sudo pacman -S dfu-util minicom
+    sudo pacman -S dfu-util minicom
     ```
 
 ### Flashing
 
 Steps to flash firmware to the Twonkie:
 
 1. Download the latest firmware from [the project's release page](https://github.com/dojoe/Twonkie/releases)
-1. While holding the SW1 button on the Twonkie, connect it to your computer using
-   a USB micro-B cable
+   In our case, for Twonkie v2.0
+1. While **holding the SW1 button** on the Twonkie, connect it to your computer
+   using a USB micro-B cable
 1. Verify in `lsusb` that the device has booted in DFU mode:
 
     ```bash
-    $ lsusb | grep 0483:df11
+    lsusb | grep 0483:df11
     Bus 003 Device 083: ID 0483:df11 STMicroelectronics STM Device in DFU Mode
     ```
 
+    > If not detected, the button was probably not pressed.
+
 1. Navigate to the location where you downloaded the firmware
 1. Flash the firmware:
 
+    > The command may take a long time to complete (~30 minutes). Do not be
+    > alarmed if progress appears to stop. The percentage counting should
+    > start after ~5 minutes.
+
     ```bash
-    $ sudo dfu-util -a 0 -s 0x08000000 -D ~/Downloads/twonkiev2-20230611.bin
+    sudo dfu-util -a 0 -s 0x08000000 -D ~/Downloads/twonkiev2-20230611.bin
     ```
 
-    > The command may take a long time to complete. Do not be alarmed if progress
-    > appears to stop.
+    > If u see ```DFU state(10) = dfuERROR, status(10) = Device's firmware is```
+    ```corrupt. It cannot return to run-time (non-DFU) operations```
+    > this is issue with dfu-util, Unfortunately, it's normal, flashing is
```

Comment:
I don't think it's actually an issue with dfu-util, it's a status message from STM describing why DFU mode was entered

Reply:
Done: df0c124778f253bf37e494de16c287cba3b2d653
Replaced by:
```
    > If you see ```DFU state(10) = dfuERROR, status(10) = Device's firmware is```
    ```corrupt. It cannot return to run-time (non-DFU) operations```
    > this is a status message from STM describing why DFU mode was entered.
    > Flashing is probably working properly
```

Reply:
Thanks